src/arrays/creating_arrays/hard/q2/RevdArray.java

```diff
@@ -0,0 +1,48 @@
+package arrays.creating_arrays.hard.q2;
+
+/*
+Write a program asking the user for ten numbers and stores them in an integer array. Then use a method called reverse
```

Comment:
I don't know if we can have this question where the method includes dealing with an array here because it is before the arrays with methods section.